assets/css/main.css

```diff
@@ -0,0 +1,996 @@
+    background-color: #f4f5f7;
+    border-top: 1px solid #e2e2e2;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+  }
+}
+
+@media (min-width: 768px) {
+  h3 {
+    font-size: 24px;
+    line-height: 24px;
+  }
+
+  .magazine h5 {
+    font-size: 20px;
+    line-height: 20px;
+  }
+
+  .order-2 {
+    order: -1;
+  }
+
+  .culture-item {
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  .culture-stories {
+    padding-top: 0 !important;
+  }
+
+  .culture-items {
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  .section-picks article.py-3 {
+    padding-bottom: 0 !important;
+    padding-top: 0 !important;
+  }
+
+  .form-img img {
+    height: 100%;
+  }
+
+  .subscribe-block .f1 {
+    width: 80%;
+    text-align: left;
+  }
+
+  .subscribe-block .col-l .image.last {
+    padding-right: 0 !important;
+    padding-bottom: 0;
+  }
+
+  .col-lg-6.border-bottom {
+    border-bottom: 0 !important;
+  }
+
+  .magazine .n-title {
+    border-bottom: 1px solid #d9d9d9;
+    padding-bottom: 8px;
+  }
+
+  .subscribe-block .col-l .image {
+    padding-right: 25px;
+  }
+
+  .section-picks .block-title {
+    border-top: 0;
+  }
+
+  .subscribe-block {
+    position: relative;
+  }
+
+  .subscribe-block::before {
+    content: "";
+    display: block;
+    background-color: #f4f5f6;
+    height: 2px;
+    position: absolute;
+    top: 0;
+    left: -15px;
+    right: -15px;
+  }
+
+  .section-picks ul li:last-of-type {
+    border-bottom: 0;
+  }
+
+  .page-footer .footer-links li:not(:last-of-type) {
+    border-right: 1px solid #d9d9d9;
+    padding-right: 10px;
+    margin-right: 10px;
+  }
+
+  .article-summary {
+    font-size: 1em;
+  }
+
+  .section-picks h4 {
+    font-size: 1.2rem;
+  }
+}
+
+.subscribe-block .col-l .image {
+  position: relative;
+  padding-bottom: 14px;
+  padding-right: 90px;
+}
+
+.nav-header .header-top.fixed {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+  background-color: #f72210;
+  height: 50px;
+  align-items: center;
+  padding: 0 10px;
+  transition: all 0.4 ease-in-out;
+  z-index: 99999;
+}
+
+.nav-header .fixed .header-logo .brand svg {
+  width: 132px;
+  height: 20px;
+  margin: 0;
+}
+
+@media (max-width: 992px) {
+  .nav-header .container-fluid {
+    margin-top: 4rem;
+    padding: 0;
+  }
+
+  .nav-header .header-top {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    width: 100%;
+    background-color: #f72210;
+    height: 50px;
+    padding: 0 10px;
+    z-index: 99999;
+  }
+
+  .nav-header .header-logo .brand svg {
+    width: 132px;
+    height: 20px;
+  }
+
+  .nav-header .header-react {
+    padding-bottom: 0.5rem;
+  }
+}
+
+@media (min-width: 993px) {
+  .container-fluid {
+    max-width: 1310px;
+    padding: 0 30px;
+  }
+
+  .nav-header .container-fluid {
+    padding-left: 0 !important;
+    padding-right: 0 !important;
+  }
+
+  .d-pt {
+    padding-top: 1.5rem !important;
+    padding-bottom: 1.5rem !important;
```

Comment:
`pt-lg-2 pb-lg-2` --> bootstrap classes

Reply:
Done!

Reply:
**Changes made to requests above:**
   - Fix HTML validation error
   - Use bootstrap classes to reduce the number of !important keyword used.